recipes/tableone/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "tableone" %}
+{% set version = "0.4.8" %}
+{% set sha256 = "63c6d55a7bc0010ced0a6b1e88f295ce41fd93d9f8a473e14cf1c760c93a27b1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  # noarch: python
```

Comment:
This looks like it could be a noarch package

Reply:
I thought it might be noarch, but wasn't 100% sure. I read somewhere that this requires compatibility with ~all versions of Python, so thought it safer to omit. Will add for now and will clean up the build packages. Thanks again for your advice!

Reply:
It does, kinda. If you don't support py2k you can set the build/run python to `python >2`.